src/components/apis/list-of-apis/listOfApisHandlers.ts

```diff
@@ -68,27 +70,43 @@ export class ListOfApisTilesHandlers implements IWidgetHandler {
 
         return widgetOrder;
     }
+
     public getStyleDefinitions(): StyleDefinition {
         return {
+            colors: {
+                borderColor: {
+                    displayName: "Search input border color",
+                    defaults: {
+                        value: "#505050"
+                    }
+                },
+                tagButtonColor: {
+                    displayName: "Tag button color",
+                    defaults: {
+                        value: "#555"
+                    }
+                }
+            },
             components: {
                 listOfApisTiles: {
                     displayName: "List of APIs Tiles",
                     plugins: ["margin", "padding", "background", "typography"],
                     components: {
-                        searchInput: DefaultStyleDefinitions.SearchInput,
-                        apiCard: DefaultStyleDefinitions.Card,
-                        cardTitle: DefaultStyleDefinitions.CardTitle,
-                        cardText: DefaultStyleDefinitions.CardText,
-                        widgetText: DefaultStyleDefinitions.WidgetText,
-                        tagInput: DefaultStyleDefinitions.TagInput,
-                        toggleButtonLabel: DefaultStyleDefinitions.ToggleButtonLabel,
-                        tagCard: DefaultStyleDefinitions.TagCard
+                        searchInput: cloneDeep(DefaultStyleDefinitions.SearchInput),
```

Comment:
Can you describe why cloneDeep is needed?

Reply:
When modifying the style definitions for a widget, the object gets modified. So, without cloneDeep, all the widgets that have the same style definition will be modified:
![clonedeep](https://user-images.githubusercontent.com/92857141/169785595-886f9047-83b8-446e-8005-2d31779c6aa7.gif)